WooCommerce/WooCommerceUITests/Screens/Login/PrologueScreen.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+import XCTest
+
+private struct ElementStringIDs {
+    static let continueButton = "Prologue Continue Button"
+    static let siteAddressButton = "Prologue Self Hosted Button"
+}
+
+class PrologueScreen: BaseScreen {
```

Comment:
nit: (same as above)
```suggestion
final class PrologueScreen: BaseScreen {
```